packages/tackle-box/lib/svg.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite-plugin-svgr/client" />
```

Comment:
Is this what fixed the import error message?

Reply:
Nah, this was to get SVGs working. The thing that fixed the import error message is here: https://github.com/iron-fish/mobile-wallet/pull/72/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357R7